From c5f1a58582c46f2eaac727c170e0645d220abc76 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 6 Apr 2022 11:42:37 +0200
Subject: Workaround: do not depend on infra/orchestrator

Official tarballs depend on third_party/blink/tools, that is excluded
in the tarball creation, so GN configuration is broken.

This has been reported as https://crbug.com/1313361 but, while it is
not fixed, let's workaround it.

Bug: 1313361
Change-Id: Iece0a63ed0b8e479bb92980ed98031fb8605b5b3
---
 BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/BUILD.gn b/BUILD.gn
index 73aefc2325553..167deea13f789 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -86,7 +86,6 @@ group("gn_all") {
     "//codelabs",
     "//components:components_unittests",
     "//components/gwp_asan:gwp_asan_unittests",
-    "//infra/orchestrator:orchestrator_all",
     "//net:net_unittests",
     "//services:services_unittests",
     "//services/service_manager/public/cpp",
-- 
2.32.0

